sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -420,8 +420,12 @@ case class FileSourceScanExec(
       selectedPartitions: Seq[PartitionDirectory],
       fsRelation: HadoopFsRelation): RDD[InternalRow] = {
     val openCostInBytes = fsRelation.sparkSession.sessionState.conf.filesOpenCostInBytes
-    val maxSplitBytes =
-      FilePartition.maxSplitBytes(fsRelation.sparkSession, selectedPartitions)
+    val maxSplitBytes = relation.fileFormat match {
+      case _ : ParquetSource =>
+        fsRelation.sparkSession.sessionState.conf.filesMaxPartitionBytes // parquet.block.size
```

Comment:
shouldn't this be something provided by the ParquetSource?

Reply:
Actually, `ParquetSource` here is alias of `ParquetFileFormat`
```scala
import org.apache.spark.sql.execution.datasources.parquet.{ParquetFileFormat => ParquetSource}
```

Reply:
Sorry, please ignore. Currently, `FileFormat` doesn't provide this. How to split is determined in `DataSourceScanExec`